src/Root.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Root of the React component tree.
+ * @flow
+ */
+
+var React = require('react'),
+    Controls = require('./Controls'),
+    GenomeTrack = require('./GenomeTrack'),
+    types = require('./types'),
+    // TODO: make this an "import type" when react-tools 0.13.0 is out.
+    TwoBit = require('./TwoBit');
+
+var Root = React.createClass({
+  propTypes: {
+    referenceSource: React.PropTypes.object.isRequired
+  },
+  getInitialState: function(): any {
+    return {
+      contigList: ['1', '2', '3'],
```

Comment:
Any reason this isn't just blank, as it'll be filled in by `componentDidMount`?  Flash of no-content type consideration?